tests/integration/test_cli.py

```diff
@@ -221,26 +231,78 @@ def test_upload(
 
 
 @pytest.mark.parametrize(
-    "bad_url,file_id,expected_exception",
+    "file_size,anticipated_part_size",
     [
-        (False, "uploaded", None),
-        (False, "uploadable", BadResponseCodeError),
-        (True, "uploaded", RequestFailedError),
+        (6 * 1024 * 1024, 5),
+        (20 * 1024 * 1024, 16),
     ],
 )
-def test_confirm_api_call(
-    bad_url,
-    file_id,
-    expected_exception,
+def test_multipart_upload(
```

Comment:
The 5 MiB part size is the minimum possible part size, thats why we test for that seperately.
We don't test for bigger part numbers, because we have to inject all presigned urls into the container and I don't see, how we would gain anything from raising that number.

Reply:
Sure, thanks for claryfying, had to check with the mock_api again to get the context and wanted to make sure I understood correctly